kubernetes/assets/recommended_monitors/monitor_pod_memory_limit.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "[kubernetes] Monitor Kubernetes Pod Memory Limit Reached",
```

Comment:
same comments as first monitors.

measure the number of OOMs instead. If OOMs is very often, it could trigger crashloopbackoff.